runtime/vm/resolvesupport.cpp

```diff
@@ -2297,17 +2297,21 @@ resolveInvokeDynamic(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA callSite
 			result = vmThread->currentException;
 		} else if (result == NULL) {
```

Comment:
The comparisons here should be inverted.

Reply:
inverted.